src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -439,29 +463,30 @@ DOCKER_OPTS=" -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock"
 http://localhost:2375/info
 ----
 
-Having verified that your Docker daemon is listening, run the Che container with the with `DOCKER_HOST` environment variable set to the IP address of `docker0` or `eth0` network interface. If `docker0` is running on 1.1.1.1 then:
+After you have verified that your Docker daemon is listening, run the Che container with `DOCKER_HOST` environment variable set to the IP address of `docker0` or `eth0` network interface. If `docker0` is running on 1.1.1.1:
 
 ----
 docker run -ti -e DOCKER_HOST=tcp://1.1.1.1:2375 -v /var/run/docker.sock:/var/run/docker.sock -v ~/Documents/che-data1:/data eclipse/che start
 ----
 
-Alternatively, you can save this env in `che.env` and restart Che.
+Alternatively, you can save this environment variable in the `che.env` file and restart Che.
 
 [id="proxiesfirewallsports"]
-== Proxies/Firewalls/Ports
+== Proxies/firewalls/ports
+
+To install and operate Che behind a proxy, take the following steps:
 
-You can install and operate Che behind a proxy:
+. Configure each physical node’s Docker daemon with proxy access.
 
-1.  Configure each physical node’s Docker daemon with proxy access.
-2.  Optionally, override workspace proxy settings for users if you want to restrict their Internet access.
+. Optionally, override workspace proxy settings for users if you want to restrict their Internet access.
 
-Before starting Che, configure https://docs.docker.com/engine/admin/systemd/#/http-proxy[Docker’s daemon for proxy access]. If you have Docker for Windows or Docker for Mac installed on your desktop and installing Che, these utilities have a GUI in their settings which let you set the proxy settings directly.
+. Before starting Che, configure https://docs.docker.com/engine/admin/systemd/#/http-proxy[Docker’s daemon for proxy access]. If you have Docker for Windows or Docker for macOS installed on your desktop and installing Che, these utilities have a GUI in their settings which let you set the proxy settings directly.
 
-Please be mindful that your `HTTP_PROXY` and/or `HTTPS_PROXY` that you set in the Docker daemon must have a protocol and port number. Proxy configuration is quite finnicky, so please be mindful of providing a fully qualified proxy location.
+. Ensure that your `HTTP_PROXY` and/or `HTTPS_PROXY` that you set in the Docker daemon have a protocol and port number. Proxy configuration can be problematic; hence, ensure that you provide a fully qualified proxy location.
```

Comment:
* no slash
* Proxy configuration can be problematic; -- redundant
* no 'hence'